templates/federated_reporting/import.sh

```diff
@@ -90,6 +93,22 @@ else
   done
 fi
 
+if [ "$CFE_FR_HANDLE_DUPLICATES" = "yes" ]; then
+  log "Handle Duplicate Hostkeys"
+  hostkey_list=$(printf "'%s'," ${hostkeys[*]} | sed -e 's/,$//')
+  "$CFE_BIN_DIR"/psql -U $CFE_FR_DB_USER -d cfdb --set "ON_ERROR_STOP=1" \
+                      -c "SELECT handle_duplicate_hostkeys_in_import(ARRAY[$hostkey_list]);"  \
+    > duplicates.log 2>&1 || failed=1
+  if [ "$failed" = "0" ]; then
+    log "Handle Duplicate Hostkeys: DONE"
+  else
+    log "Handle Duplicate Hostkeys: FAILED"
+    log "last 10 lines of duplicates.log"
```

Comment:
Where is this `dupliactes.log`?

I think we need some documentation about this, in MPF.md and perhaps in a MD living inside the FR directory, that we can just pull into the docs site. It seems like this log file could grow unbounded, so we need to highlight that, or add it to our log files that are managed.

Reply:
Let's not do it in this cherry-pick, though.

Reply:
Added a checkbox todo on ENT-6035.

Reply:
> Where is this `dupliactes.log`?
> 
> I think we need some documentation about this, in MPF.md and perhaps in a MD living inside the FR directory, that we can just pull into the docs site. It seems like this log file could grow unbounded, so we need to highlight that, or add it to our log files that are managed.

This `duplicates.log` file won't grow very large. Unless there are errors it will be a constant size regardless of the number of hosts or duplicates found. It is also written to with `<` so will always be written over on each import.